pkg/lvm/lvm_util.go

```diff
@@ -254,6 +256,97 @@ func ResizeLVMVolume(vol *apis.LVMVolume, resizefs bool) error {
 	return err
 }
 
+// ResizeBTRFSVolume resizes the LVM volume as well as expand btrfs filesystem
+// Step1: Expanding LVM Volume (lvextend lvmvg -n <volume_name> -L size)
+// Step2: Expanding btrfs filesystem (btrfs filesystem resize max <mount_path>)
+func ResizeBTRFSVolume(vol *apis.LVMVolume, mountPath string) error {
```

Comment:
let us move this function to https://github.com/openebs/lib-csi/tree/develop/pkg/btrfs, so that other localpv can use it.

Reply:
we might need to refactor this function and remove LVM related code.

Reply:
Raised a [PR](https://github.com/openebs/lib-csi/pull/10) in lib-csi repo.